src/app/(app)/home/entries/[slug]/viewer.module.scss

```diff
@@ -0,0 +1,46 @@
+.viewerHeader {
+  margin-bottom: 1rem;
+}
+
+.title {
+  font-size: 2.5rem;
+}
+
+.body {
+  color: var(--textLighter);
+  line-height: 1.75;
+}
+
+.viewerDetails {
+  margin-bottom: 2rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  .createdTimeDate {
+    display: flex;
+    flex-direction: row;
+    gap: 0.75rem;
+    color: var(--accent);
+  }
+
+  .edit {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    gap: 0.5rem;
+  }
+
+  .editButton {
+    font-weight: 500;
+    text-transform: capitalize;
+  }
+
+  .editedText {
+    opacity: 0.75;
+    font-style: italic;
+    font-size: 0.9rem;
+    color: var(--textLighter);
+  }
+}
```

Comment:
Some of these styles seem like they might be useful for other components. Which of these do you think could be useful? and if you think they could be which file would you think would be a good place to for them?

Reply:
hmm I can move some of the text styling classes to layout.module.scss! the title and body classes especially.. the little italics class might be useful too

Reply:
For our global classes we are going to want to have those in a top level css file rather than a parent scss file due to compilation behaviors. While we could have all our global classes in layout.module.scss Ive found it makes maintaining the file a little messy